example/Example.res

```diff
@@ -17,16 +17,16 @@ module Form = {
           "firstName": "",
           "lastName": "",
           "acceptTerms": false,
-          "hobbies": [{"id": Uuid.v4(), "name": ""}],
+          "hobbies": [{"name": ""}],
         }),
         (),
       ),
       (),
     )
 
-    let (hobbiesAreShown, setHobbiesAreShown) = React.useState(() => false)
+    let (hobbiesAreShown, setHobbiesAreShown) = React.useState(() => true)
 
-    let {fields, append, update} = Hooks.ArrayField.use(.
```

Comment:
`update`'s behavior is a bit... weird. I replaced it with `setValue` and it works like a charm.